runtime/moonbase/tests/integration_test.rs

```diff
@@ -31,6 +35,39 @@ use parachain_staking::{Bond, NominatorAdded};
 use sp_core::{Public, H160, U256};
 use sp_runtime::DispatchError;
 
+#[test]
+fn verify_pallet_indices() {
+	fn is_pallet_index<P: 'static>(index: usize) {
```

Comment:
Could make this generic over the runtime and put it in `common` module, am considering it. I would just add a generic `Runtime: frame_system::Config`.

Reply:
Common module isn't shared between all integration tests. There is no real point to doing this, it would just move it elsewhere so it takes longer for someone to look up the definition of this function...

Reply:
Agree. And I could also imagine cases where a pallet is installed at one index on one runtime and a different index on another runtime.